pageserver/src/virtual_file/io_engine.rs

```diff
@@ -224,6 +240,7 @@ impl IoEngine {
             #[cfg(target_os = "linux")]
             IoEngine::TokioEpollUring => {
                 // TODO: ftruncate op for tokio-epoll-uring
+                // Don't forget to use retry_ecanceled_once
```

Comment:
Did you forget?

Reply:
It only becomes relevant once we switch set_len to actually use tokio-epoll-uring => https://github.com/neondatabase/neon/issues/11817